gen/gen.go

```diff
@@ -16,6 +16,7 @@ import (
 	"github.com/filecoin-project/specs-actors/v5/actors/builtin/system"
 	"github.com/filecoin-project/specs-actors/v5/actors/builtin/verifreg"
 	"github.com/filecoin-project/specs-actors/v5/actors/util/smoothing"
+	vm_test "github.com/filecoin-project/specs-actors/v5/support/vm"
```

Comment:
This is weird. I see that the existing vm is all under this vm_test package, but I don't think that's helpful (we should reserve that for any testing _of_ the vm). Fixed in #1409.